code-studio/src/js/levelbuilder.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file Main entry point for scripts used on all level editing pages.
+ */
+'use strict';
+var _ = require('lodash');
+
+window.levelbuilder = window.levelbuilder || {};
+_.extend(window.levelbuilder, {
+  initializeCodeMirror: require('./initializeCodeMirror')
+});
+
+// TODO: Remove when global `CodeMirror` is no longer required.
+window.CodeMirror = require('codemirror');
+// TODO: Remove when global `marked` is no longer required.
+window.marked = require('marked');
```

Comment:
Yeah, that [happens here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/_apps_dependencies.html.haml#L45).  We'll work on that separately, I think.